libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -32,43 +34,68 @@ export class DefaultConfigurationSettingsConverter implements ConfigurationSetti
   public Convert(
     config: object,
     options: SourceOptions
-  ): SetConfigurationSettingParam<string | FeatureFlagValue | SecretReferenceValue>[] {
+  ): SetConfigurationSettingParam<string | MsFeatureFlagValue | SecretReferenceValue>[] {
     let configurationSettings = new Array<
-      SetConfigurationSettingParam<string | FeatureFlagValue>
+      SetConfigurationSettingParam<string | MsFeatureFlagValue>
     >();
-    let featureFlagsConfigSettings = new Array<SetConfigurationSettingParam<FeatureFlagValue>>();
-    let foundFeatureManagement = false;
-    let featureFlags: any = {};
-    let featureManagementKeyWord = "";
-    let enabledForKeyWord = "";
+
+    let featureFlagsConfigSettings = new Array<SetConfigurationSettingParam<MsFeatureFlagValue>>();
+    let foundMsFmSchema = false;
+    let foundLegacySchema = false;
+    let legacySchemaFeatureManagementKeyWord = "";
```

Comment:
Can we avoid using the term "legacy"? As time passes by, no one knows what legacy is. Please use specific names instead.

Reply:
> Do we have specific name for the old schema? Seems it's not make much sense to call it dotnetFeatureManagement.

It is a schema that was/is used by the .NET Feature Management library.

Reply:
Java was/is also using this schema, and if we call it dotnetFeatureManagement, also easier leading to confusion that it's a schema used by .net (implicit .net does not use another)

Reply:
Java Spring uses something similar but in Yaml format.